### DISCUSSION TITLE
Jira da Atlassian

### DISCUSSION THREAD

**Discussion by bomffas** [frontendbr/forum#627] [Perguntas e Respostas]
Olá Pessoal da Comunidade,

Gostaria de saber de quem usar o Jira como era o fluxo de trabalho antes e depois do uso e se houver muita dificuldade de usa-lo.

Value pessoal

**Comment by williamcosta** [comment]
@bfmjr já trabalhei em empresas que não usavam e hoje trabalho em um empresa que usa.

Posso dizer que facilita bastante para você saber o que é preciso desenvolver no projeto, quais as suas tarefas, quais tarefas dependem da sua e vice-versa. Enquanto nas empresas que trabalhei que não usavam era difícil ter essa organização.

Sobre dificuldades, é só a adaptação, a ferramenta em si não é difícil de usar ou nada parecido.

Sem contar nas N integrações que a ferramenta tem por exemplo com Bitbucket, Slack, Confluence etc.

**Comment by guilhermejcgois** [comment]
Fala @bfmjr , blz?

Comecei esse ano em uma startup onde usamos Jira integrado com Bitbucket e Slack, nunca tinha mexido com Jira até então. Achei uma puta ferramenta e achei tranquilo aprender a mexer. Recentemente eles atualizaram a interface, como estava acostumado com a antiga achei um pouco chatinha (apesar de mais bonita).

Abs,

**Comment by yanmagale** [comment]
Acho  que o workflow dentro dele vai depender muito de como sua empresa trabalha, se usam SCRUM ou Kanban ... Mas é uma ferramenta bem legal.

**Comment by mvfsillva** [comment]
Usava na empresa que trabalhava anteriormente, o bacana é que você consegue configurar totalmente o workflow, por exemplo, não permitir que um dev pegue duas tarefas simutanealmente, além de ser ótimo para quem gosta de trabalhar com branches, existem diversas integrações bem bacanas. Em termos de UX a ferramenta peca bastante, mas nada que depois de algumas semanas não dê pra se acostumar e encontrar suas features escondidas em seus múltiplos menus;
